src/class-wpml-elementor-translatable-nodes.php

```diff
@@ -903,6 +903,24 @@ public static function get_nodes_to_translate() {
 					),
 				),
 			),
+			'galleries'           => array(
+                'conditions' => array( self::TYPE => 'gallery' ),
+                'fields' => array(
+                    array(
+                        'field' => 'show_all_galleries_label',
+                        'type' => __('All Gallery Label:', 'sitepress'),
```

Comment:
One last thing @GeorgeBotsev, I don't think we add a `:` at the end of our labels for other widgets. Can you please also remove it?

Reply:
Ok, removing those.